tutorials/5 All QRenderers/5.1 Ansys/Ansys QRenderer.ipynb

```diff
@@ -240,15 +259,15 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can optionally indicate with the optional parameters whether you intend to open and use a previously saved project. <br>\n",
+    "You can also indicate, using the optional parameters, whether you intend to open and use a previously saved project. <br>\n",
     "Make sure that the saved project contains at least one design, or the method will produce a warning"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "#### 2.4 Setup an Ansys HFSS design\n",
+    "#### 2.3 Setup an Ansys HFSS design\n",
     "You can either create a new design or select and use an old one.\n",
     "\n",
     "To **create** a new design **manually**, go to the Ansys GUI and follow these instructions:\n",
```

Comment:
```
To **create** a new eigenmode design **automatically**, execute the following cell<br>
The design will be added to the project that was active when the command `fourq_hfss.connect_ansys()` was executed.
Note: If a design named `HFSSTransmonQubit` already exists in the project, a new design will be created, with the name suffixed with an incremental integer: `HFSSTransmonQubit1`, `HFSSTransmonQubit2`, etc.

Suggest:
The cell further below explains how methods activate_eigenmode_design() and activate_drivenmodal_design()  can also add a design to the active project.  The difference is that if the design exists, the integer suffix will not be added or incremented.   The design will be activated.
```

Reply:
no reason to write here what is already written few lines below.